types/textdiff-create/textdiff-create-tests.ts

```diff
@@ -0,0 +1,26 @@
+import textDiffCreate = require('textdiff-create');
+import type { Change } from 'textdiff-create';
+
+// $ExpectType Change[]
+const result = textDiffCreate('one', 'two');
+
+// $ExpectType Change
+const firstChange = result[0];
+
+switch (firstChange[0]) {
+    case textDiffCreate.ChangeType.DELETE: {
```

Comment:
the package does not export this symbol (and other costs), the types in this PR adds this. which means, in case of TS > JS migration, this will no longer works.

Reply:
right, I see. I have switched now to just using number literals.